src/start/RealmScreen.js

```diff
@@ -90,7 +83,7 @@ class RealmScreen extends React.Component {
               autoCapitalize="none"
               placeholder="Server URL"
               defaultValue={realm}
-              onChangeText={value => this.setState({ realm: value })}
+              onChangeText={value => this.setState({ realm: fixRealmUrl(value) })}
```

Comment:
surprising behaviour 
![ezgif com-video-to-gif-7](https://user-images.githubusercontent.com/18511177/27212517-458b6704-527e-11e7-9dae-fb482ef0f4ec.gif)